ietf/settings.py

```diff
@@ -1010,7 +1010,7 @@ def skip_unreadable_post(record):
 TZDATA_ICS_PATH = BASE_DIR + '/../vzic/zoneinfo/'
 
 SECR_BLUE_SHEET_PATH = '/a/www/ietf-datatracker/documents/blue_sheet.rtf'
-SECR_BLUE_SHEET_URL = '//datatracker.ietf.org/documents/blue_sheet.rtf'
+SECR_BLUE_SHEET_URL = IDTRACKER_BASE_URL + '/documents/blue_sheet.rtf'
```

Comment:
fyi - no action requested:
reverse won't work here because of init order considerations.
If we were going to keep the function that used this setting, I would suggest going to where SECR_BLUE_SHEET_URL is used and rewriting things so that reverse could be used there, but we're going to be removing this function.